planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -262,6 +262,11 @@ class PlanningSceneMonitor : private boost::noncopyable
     return current_state_monitor_;
   }
 
+  CurrentStateMonitorPtr& getStateMonitorNonConst()
```

Comment:
Yes, in order to enable the dynamics copy during run time I need access to a non-const version of the class